chainweb.cabal

```diff
@@ -348,7 +349,7 @@ library
         , swagger2 >= 2.2
         , temporary >= 1.3
         , text >= 1.2
-        , these >= 0.7 && < 1
+        , these >= 0.7 && < 1.1
```

Comment:
The `semialign` dep effectively forces `these >= 1`.

Reply:
Oh, right.  I'll make that change.